engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/misc/datafactory/DataFactoryScriptingSupport.java

```diff
@@ -448,6 +450,19 @@ public void initialize( final DataFactory dataFactory, final DataFactoryContext
     if ( globalScriptContext != null ) {
       return;
     }
+    if ( StringUtils.isEmpty( globalScriptLanguage ) ) {
+      return;
+    }
+    boolean allowScriptEval = ClassicEngineBoot.getInstance().getGlobalConfig().getConfigProperty(
+                    "org.pentaho.reporting.engine.classic.core.allowScriptEvaluation", "false" )
+            .equalsIgnoreCase( "true" );
+
+    if ( !allowScriptEval ) {
+      DataFactoryScriptingSupport.logger.error( "Scripts are prevented from running by default in order to avoid"
+              + " potential remote code execution.  The system administrator must enable this capability by changing"
+              + " the value of org.pentaho.reporting.engine.classic.core.allowScriptEvaluation to true." );
+      return;
+    }
```

Comment:
I think the merge operation is breaking something here and GitHub can't correctly calculate what to merge.